watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -104,9 +104,22 @@ def test_parameter_block(model):
     assert isinstance(model.fs.properties.solvent_set, Set)
     for j in model.fs.properties.solvent_set:
         assert j in ["H2O"]
+    # solute set and ion set are identical when all components in component_list are ions given hierarchy
+    assert isinstance(model.fs.properties.solute_set, Set)
+    for j in model.fs.properties.solute_set:
+        assert j in ["A", "B", "C", "D"]
     assert isinstance(model.fs.properties.ion_set, Set)
-    for j in model.fs.properties.ion_set | model.fs.properties.solute_set:
```

Comment:
@lbibl I think this specific testing was outdated with new set hierarchy, although it still passed because `ion_set == solute_set` in this specific example, meaning the union of the two was still passing. Double check these additional tests for components being organized by the appropriate set when not explicitly specified in `type` argument.